exla/test/exla/defn_expr_test.exs

```diff
@@ -2342,7 +2342,7 @@ defmodule EXLA.DefnExprTest do
   end
 
   describe "slicing" do
-    defn slice1(t), do: Nx.slice(t, [0, 6, 2], [2, 1, 3])
+    defn slice1(t), do: Nx.slice(t, [Nx.tensor(0), Nx.tensor(6), Nx.tensor(2)], [2, 1, 3])
```

Comment:
Instead of changing it, we should probably dup the tests. We also need tests of dynamic indexes with strides.